game-core/src/main/java/games/strategy/triplea/oddsCalculator/ta/OddsCalculatorPanel.java

```diff
@@ -127,7 +125,6 @@
   private String defenderOrderOfLosses = null;
   private Territory location = null;
   private JList<String> territoryEffectsJList;
-  private final WidgetChangedListener listenerPlayerUnitsPanel = () -> setWidgetActivation();
```

Comment:
Feel free to ignore, as this may be scope creep, but there's an identical situation in the `PlayerUnitsPanel` nested class further down in this file.